File: det of matrix.py
import numpy as np
import matplotlib.pyplot as plt
t = np.arange(1, 10)
x = np.sin(2 * np.pi * t)
y = np.cos(2 * np.pi * t)
fig, axs = plt.subplots(1, 2)
axs[0].plot(t, x)
axs[0].set_title('Sine Wave')
axs[1].plot(t, y)
axs[1].set_title('Cosine Wave')
plt.tight_layout()
plt.show()

File: det of matrix.m
%x = linspace(0, 2*pi, 100);
%y = sin(x);
%plot(x, y);
%figure;
x=linspace(0,2*pi,100000000000000);
a=cos(2*x);
b=sin(2*x);
c=2*sin(x);
plot(x,a+b+c);